plant/notifications/api/views.py

```diff
@@ -0,0 +1,21 @@
+from django.shortcuts import render
+from rest_framework.mixins import ListModelMixin
+from rest_framework.viewsets import GenericViewSet
+
+from plant.notifications.api.serializers import NotificationSerializer
+from plant.notifications.models import Notification
+
+
+# Create your views here.
+
+class NotificationListView(ListModelMixin,GenericViewSet):
+    queryset = Notification.objects.all()
+    serializer_class = NotificationSerializer
+
+class NotificationUserListView(ListModelMixin,GenericViewSet):
```

Comment:
Endpoint for listing notifications should have at least 3 tests - for user listing his own notifications, for user trying to fetch notifications of other user and for unauthorized attempt